drivers/regulator/Kconfig.rpi_pico

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config REGULATOR_RPI_PICO
```

Comment:
The description and help string of this Kconfig need to be updated

Reply:
Thank you for reviewing.
I updated the description.